examples/kinsol_ors.py

```diff
@@ -18,8 +18,9 @@
 import os
 import nose
 import numpy as np
-import scipy as sp
 import scipy.sparse as sps
+import scipy.sparse.linalg as spsl
```

Comment:
Is there any particular motivation for this change?

Reply:
yes, scipy.sparse.linalg is a standalone module thats not imported by scipy.sparse so the following does not work:
```
import scipy.sparse as sps
# cannot use sps.linalg
```
instead we must import scipy.sparse.linalg separately